orx/grantlee/0.2/OrxJs.qs

```diff
@@ -93,8 +101,8 @@ function normPointToString(point) {
         return 'top|right';
     if (point.x == 0.0 && point.y == 1.0)
         return 'bottom|left';
```

Comment:
The literals should be separated by spaces, ie. `top left` instead of `top|left`.